model/src/pyrenew/latent/infection_seeding_method.py

```diff
@@ -0,0 +1,51 @@
+from abc import ABCMeta, abstractmethod
+
+import jax.numpy as jnp
+from jax.typing import ArrayLike
+
+# -*- coding: utf-8 -*-
+# numpydoc ignore=GL08
+"""
+Classes for the infection seeding methods
+"""
+
+
+class InfectionSeedMethod(metaclass=ABCMeta):
+    def __init__(self, n_timepoints: int):
```

Comment:
I'm not sure if `InfectionSeedMethod` should have an `I0` attribute or how to implement sampling, if it does.

Reply:
Don't do that.